tests/staticfile.py

```diff
@@ -0,0 +1,13 @@
+import nose
+import requests
+
+import fixture
+
+
+@nose.with_setup(fixture.start_tangelo, fixture.stop_tangelo)
+def test_static_file():
+    r = requests.get(fixture.url("static_file"))
+    print r.text
+    assert r.ok
+    assert r.status_code == 200
+    assert r.text == "Infinite Diversity in Infinite Combinations\n"
```

Comment:
We might as well test the mime type pass-though aspect of this.

For instance, you could add to the end of this file:
```
    assert "text/plain" in r.headers["Content-Type"]
    r = requests.get(fixture.url(
        "static_file?mime_type=application/octet-stream"))
    assert r.ok
    assert r.status_code == 200
    assert r.text == "Infinite Diversity in Infinite Combinations\n"
    assert "application/octet-stream" in r.headers["Content-Type"]
```
and alter `tests/web/static_file.py` to be
```
def run(**kwargs):
    return tangelo.file("../static_file.txt",
                        content_type=kwargs.get("mime_type", "text/plain"))
```